apps/account-api/src/controllers/v1/keys-v1.controller.ts

```diff
@@ -76,4 +85,56 @@ export class KeysControllerV1 {
       throw new HttpException('Failed to find public keys for the given msaId', HttpStatus.BAD_REQUEST);
     }
   }
+
+  @Get('publicKeyAgreements/getAddKeyPayload')
+  @HttpCode(HttpStatus.OK)
+  @ApiQuery({
+    name: 'newKey',
+    description: 'New public key to be added in hex format',
+    type: 'string',
+    required: true,
+  })
```

Comment:
Don't need this decorator if `@Query()` parameter is a properly-decorated DTO class (see other comment)